contrib/cmd/remap-rootfs/remap-rootfs.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func usage() {
+	fmt.Println("usage: remap-rootfs <bundle>")
+	os.Exit(1)
+}
```

Comment:
Needs more documentation, as markdown or godoc

Reply:
I'll switch to `urfave/cli`, though I sometimes wonder if we need a separate place for programs that we only use for test purposes. `fs-idmap` is basically useless to anyone outside of our tests, and while `remap-rootfs` is theoretically useful I doubt anyone will use it.

Reply:
I was thinking the same too :)